middleware/auth.js

```diff
@@ -0,0 +1,90 @@
+const jwt = require("jsonwebtoken");
+const config = require("config");
+const mongoose = require("mongoose");
+const { User } = require("../models/user");
+
+userAuth = async function (req, res, next) {
```

Comment:
Why are you not using `let` or `var` or `const` to declare your variables?

Reply:
ohhh .. my bad would correct that.Thanks.